### ISSUE TITLE
Cannot import cntk on microsoft/cntk:2.2-cpu-python3.5

### ISSUE THREAD

**Issue by bigonazzi** [Microsoft/CNTK-docker#3]
Hi, 

when I try running CNTK on a CPU only machine (my laptop) I get: 
> ImportError: libnvidia-ml.so.1: cannot open shared object file: No such file or directory

To repro:
- `docker pull microsoft/cntk:2.2-cpu-python3.5`
- `docker run -d -p 8888:8888 --name cntk2.2 -t microsoft/cntk:2.2-cpu-python3.5`
- `docker exec -it cntk2.2 bash`
- `jupyter-notebook --no-browser --port=8888 --ip=0.0.0.0 --notebook-dir=/cntk/Tutorials --allow-root`
- open up notebook url and execute cell #5

I've tried the same steps with microsoft/cntk:2.1-cpu-python3.5, and everything works fine on the very same machine.

Here's the full output from CNTK_101_LogisticRegression notebook:

```
# Import the relevant components
from __future__ import print_function
import numpy as np
import sys
import os

import cntk as C
import cntk.tests.test_utils
cntk.tests.test_utils.set_device_from_pytest_env() # (only needed for our build system)
C.cntk_py.set_fixed_random_seed(1) # fix the random seed so that LR examples are repeatable
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
~/anaconda3/envs/cntk-py35/lib/python3.5/site-packages/cntk/cntk_py.py in swig_import_helper()
     17         try:
---> 18             return importlib.import_module(mname)
     19         except ImportError:

~/anaconda3/envs/cntk-py35/lib/python3.5/importlib/__init__.py in import_module(name, package)
    125             level += 1
--> 126     return _bootstrap._gcd_import(name[level:], package, level)
    127 

~/anaconda3/envs/cntk-py35/lib/python3.5/importlib/_bootstrap.py in _gcd_import(name, package, level)

~/anaconda3/envs/cntk-py35/lib/python3.5/importlib/_bootstrap.py in _find_and_load(name, import_)

~/anaconda3/envs/cntk-py35/lib/python3.5/importlib/_bootstrap.py in _find_and_load_unlocked(name, import_)

ImportError: No module named 'cntk._cntk_py'

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
<ipython-input-1-fca1ad3d7715> in <module>()
      5 import os
      6 
----> 7 import cntk as C
      8 import cntk.tests.test_utils
      9 cntk.tests.test_utils.set_device_from_pytest_env() # (only needed for our build system)

~/anaconda3/envs/cntk-py35/lib/python3.5/site-packages/cntk/__init__.py in <module>()
      8 import numpy as np
      9 
---> 10 from . import cntk_py
     11 
     12 #

~/anaconda3/envs/cntk-py35/lib/python3.5/site-packages/cntk/cntk_py.py in <module>()
     19         except ImportError:
     20             return importlib.import_module('_cntk_py')
---> 21     _cntk_py = swig_import_helper()
     22     del swig_import_helper
     23 elif _swig_python_version_info >= (2, 6, 0):

~/anaconda3/envs/cntk-py35/lib/python3.5/site-packages/cntk/cntk_py.py in swig_import_helper()
     18             return importlib.import_module(mname)
     19         except ImportError:
---> 20             return importlib.import_module('_cntk_py')
     21     _cntk_py = swig_import_helper()
     22     del swig_import_helper

~/anaconda3/envs/cntk-py35/lib/python3.5/importlib/__init__.py in import_module(name, package)
    124                 break
    125             level += 1
--> 126     return _bootstrap._gcd_import(name[level:], package, level)
    127 
    128 

ImportError: libnvidia-ml.so.1: cannot open shared object file: No such file or directory
```

**Comment by bgie** [comment]
Same here.
I created another issue before I found this one: https://github.com/Microsoft/CNTK/issues/2632

Trying the 2.1 version now...

**Comment by bgie** [comment]
This was fixed in the 2.3 version.
See https://github.com/Microsoft/CNTK/issues/2632
